cpp/data/installer/scripts/install_ubtools.sh

```diff
@@ -5,6 +5,21 @@
 # compile cpp installer
 # run it
 
+# ./install_ubtools.sh parameter
+# parameter option:
+#   - fulltext-backend  <--test | --production> [--omit-cronjobs] [--omit-systemctl] [--omit-systemctl]
+#   - ub-tools-only
+#   - vufind < ixtheo | krimdok >  <--test | --production> [--omit-cronjobs] [--omit-systemctl] [--omit-systemctl] 
+#  
+# example:
+# ./install_ubtools.sh vufind ixtheo --test --omit-cronjobs | tee ~/output.log
+
+function ColorEcho {
+    echo -e "\033[1;34m" $1 "\033[0m"
+}
+
+ColorEcho "*************** The installation script is running *******************"
```

Comment:
If you really want to add ColorEcho and add comments like this, we should use it on multiple occurences. E.g. either we divide this script into sections and call it at every section head, or we change all  echo calls to use ColorEcho instead.